vital/types/geo_point.cxx

```diff
@@ -46,48 +46,48 @@ using geo_raw_point_t = geo_point::geo_raw_point_t;
 
 // ----------------------------------------------------------------------------
 geo_point::
-geo_point()
+  geo_point()
   : m_original_crs{ -1 }
 { }
 
 // ----------------------------------------------------------------------------
 geo_point::
-geo_point( geo_raw_point_t const& point, int crs )
-  : m_original_crs( crs )
+  geo_point(geo_raw_point_t const& point, int crs)
```

Comment:
Please don't change this; the old style matches the dubiously-official style for KWIVER.